pkg/codegen/nodejs/gen.go

```diff
@@ -845,7 +845,13 @@ func (mod *modContext) genType(w io.Writer, obj *schema.ObjectType, input bool,
 	}
 
 	wrapInput := input && !mod.details(obj).functionType
-	mod.genPlainType(w, tokenToName(obj.Token), obj.Comment, properties, input, wrapInput, false, level)
+
+	name := tokenToName(obj.Token)
+	if wrapInput && mod.compatibility != tfbridge20 {
+		name += "Args"
```

Comment:
I ran this on Azure-Native - it looks like this changes the type name from `Foo` to `FooArgs`, but the parent type still tries referencing `Foo`. Example of an error:

```
web/webAppSourceControlSlot.ts:148:66 - error TS2694: Namespace '"/Users/mikhailshilkov/go/src/github.com/pulumi/pulumi-azure-native/sdk/nodejs/types/input".web' has no exported member 'GitHubActionConfiguration'.

148     readonly gitHubActionConfiguration?: pulumi.Input<inputs.web.GitHubActionConfiguration>;
```

Reply:
See https://github.com/pulumi/pulumi-azure-native/actions/runs/711650762